### PR TITLE
Fixed #32749 -- Doc'd PyMemcacheCache defaults.

### PR DESCRIPTION
**Pull request by pmontepagano**

### CODE REVIEW

File: docs/topics/cache.txt [hunk 1]
```diff
@@ -548,6 +548,17 @@ flag on the connection's socket::
         }
     }
 
+``pymemcache`` by default sets the following options (you can override them on
+your :setting:`OPTIONS <CACHES-OPTIONS>`)::
+
+    'OPTIONS': {
+        'allow_unicode_keys': True,
+        # WARNING: changing default_noreply to True is not supported.
+        # See https://code.djangoproject.com/ticket/32749#comment:3
```
Comment (felixxm) [inline docs/topics/cache.txt:557]: Changing `default_noreply` breaks our test suite, but folks in general can do this. I would drop this comment.

File: docs/topics/cache.txt [hunk 2]
```diff
@@ -548,6 +548,15 @@ flag on the connection's socket::
         }
     }
 
+``pymemcache`` by default sets the following options (you can override them on
+your :setting:`OPTIONS <CACHES-OPTIONS>`)::
```
Comment (felixxm) [inline docs/topics/cache.txt:552]: The `PyMemcacheCache` backend sets this options, not `pymemcache` itself, so maybe:
```suggestion
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::
```

File: docs/topics/cache.txt [hunk 3]
```diff
@@ -548,6 +548,15 @@ flag on the connection's socket::
...
+
+    'OPTIONS': {
+        'allow_unicode_keys': True,
+        'default_noreply': False,
+        'serde': pymemcache.serde.pickle_serde,
+    }
```
Comment (felixxm) [inline docs/topics/cache.txt:558]: I would move this to the ["Memcached" section](https://docs.djangoproject.com/en/dev/topics/cache/#memcached), just before _"A final point about...."_:
```
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::

    'OPTIONS': {
        'allow_unicode_keys': True,
        'default_noreply': False,
        'serde': pymemcache.serde.pickle_serde,
    }

A final point about...
```

**Review by felixxm — COMMENTED** [review]
@pmontepagano Thanks for this patch :+1: I left small comments.

File: docs/topics/cache.txt (hunk 3 above)
Comment (pmontepagano) [reply docs/topics/cache.txt:558]: Yes, you're right, it fits better there.

**Pull request merged**